frontend/styles/global/_global.scss

```diff
@@ -21,7 +21,8 @@ body {
 
   .__react_component_tooltip.show {
     opacity: 1; // Overrides 0.9 default opacity
-    text-align: center;
+    text-align: left;
```

Comment:
@ghernandez345 @mike-j-thomas 

Can either of you elaborate intended text-alignment for tooltips?

Reply:
according to the current state of the [style guide](https://www.figma.com/file/qbjRu8jf01BzEfdcge1dgu/Fleet-style-guide-2022-(WIP)?node-id=213%3A14506&t=HSvT3VgMx99ew9XE-0):
<img width="800" alt="Screenshot 2022-11-21 at 1 58 51 PM" src="https://user-images.githubusercontent.com/61553566/203166558-3cb7ce6d-8737-4a2c-ae8a-5e99287aa086.png">

Reply:
looks good then! Thanks for making the global fix!

Reply:
Ah... I'm sorry. This isn't entirely accurate. So there's tooltips, which refers to rolling over an icon or "tool" to say what it is. (documented in the example above). Then there's "contextual help" which is triggered by the underlined field label or text. I believe we also refer to this as tooltip in the product. This is marked "todo" in the style guide as the guidelines are a little more complicated. These usually include a longer description. Sometimes a link. And should be aligned left or right depending on their position in relation to the page edge.

Sorry for the confusion. I wanted to separate this to elements before publishing the style guide officially.

I'll make this my next priority for the style guide.

Reply:
Thanks for clarifying @mike-j-thomas. @RachelElysia since this doesn't only apply to this PR, how about I submit this as-is, and then once Mike finalizes this part of the style guide I can do an interface-wide tooltip / contextual help style refactor?

Reply:
Sorry for not responding sooner, @RachelElysia. Yes, text-align left is usually always preferable. It's better for readability.

> once Mike finalizes this part of the style guide, I can do an interface-wide tooltip / contextual help style refactor

Thanks, @jacobshandling 🔥